tgtg/__init__.py

```diff
@@ -1,16 +1,18 @@
 import datetime
+import time
 import random
 from http import HTTPStatus
 from urllib.parse import urljoin
 
 import requests
 
-from .exceptions import TgtgAPIError, TgtgLoginError
+from .exceptions import TgtgAPIError, TgtgLoginError, TgtgPollingError
 
 BASE_URL = "https://apptoogoodtogo.com/api/"
 API_ITEM_ENDPOINT = "item/v7/"
 LOGIN_ENDPOINT = "auth/v2/loginByEmail"
-SIGNUP_BY_EMAIL_ENDPOINT = "auth/v2/signUpByEmail"
+AUTH_BY_EMAIL_ENDPOINT = "auth/v3/authByEmail"
```

Comment:
```suggestion
SIGNUP_BY_EMAIL_ENDPOINT = "auth/v2/signUpByEmail"
AUTH_BY_EMAIL_ENDPOINT = "auth/v3/authByEmail"
```

Reply:
Why this endpoint should stay in this lib ? It's not used anymore.

Reply:
It is currently still used in `signup_by_email`. If you remove it there, it can be removed here, too.

Reply:
Ah yes, your right. 
But this function is never used, so it's garbage. I will remove it at the afternoon.

Reply:
Fixed!